DigitalOffice/Services/UserService/Broker/Consumers/AccessValidatorConsumer.cs

```diff
@@ -0,0 +1,31 @@
+﻿using LT.DigitalOffice.Kernel.AccessValidator.Requests;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.UserService.Repositories.Interfaces;
+using MassTransit;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.UserService.Broker.Consumers
+{
+    public class AccessValidatorConsumer : IConsumer<IAccessValidatorRequest>
+    {
+        private readonly IUserRepository repository;
+
+        public AccessValidatorConsumer(IUserRepository repository)
+        {
+            this.repository = repository;
+        }
+
+        public async Task Consume(ConsumeContext<IAccessValidatorRequest> context)
+        {
+            var request = context.Message;
+
+            var isAdmin = repository.GetUserInfoById(request.UserId).IsAdmin;
```

Comment:
Here it is necessary to catch the exception from repository

Reply:
Good point. Will do

Reply:
Done